packages/builtin/asio_event.pony

```diff
@@ -13,6 +18,18 @@ primitive AsioEvent
     """
     AsioEventID
 
+  fun get_readable(ev: AsioEventID): Bool =>
+    @pony_asio_event_get_readable(ev)
+
+  fun set_readable(ev: AsioEventID, readable': Bool) =>
+    @pony_asio_event_set_readable(ev, readable')
+
+  fun get_writeable(ev: AsioEventID): Bool =>
+    @pony_asio_event_get_writeable(ev)
+
+  fun set_writeable(ev: AsioEventID, writeable': Bool) =>
+    @pony_asio_event_set_writeable(ev, writeable')
```

Comment:
I don't think we should have these wrapper functions - the `net` package actors should be calling the FFI functions directly.

My objection is that these functions provide ways to read/write to a `AsioEventID tag`, which does not respect capabilities and memory safety.  That is, just because you've used them safely in this PR (and helpfully commenting above each usage how/why it is safe :medal_sports:), doesn't mean that users will be as careful.  And they shouldn't *have to* be careful - part of the point of Pony is being able to write code that respects the compiler rules and assume that it is concurrency safe without thinking about the issue any deeper than that.

If we make the unsafe operations be FFI-only (by removing these wrappers), than a programmer can only do an unsafe operation by doing an FFI call, which is exactly what we want.  The fact that it's an FFI call should set off alarm bells for the programmer, and make them think about carefully proving to themselves the safety of what they're doing (since they now know the compiler won't be proving the safety for them).

Removing the wrapper functions and simply calling the FFI functions directly from `TCPConnection` et al is a simple way to avoid this concern while not fundamentally changing the nature of your solution.  These actors are already using FFI extensively, so it shouldn't be a burden.